src/components/browsingContainer/index.js

```diff
@@ -0,0 +1,6 @@
+import React from 'react';
+import './style/browsingContainer.css';
+
+export default function Index({ children }) {
```

Comment:
Just name the component as it is: BrowsingContainer, in my optinion it make much more sens and is better searchable. If all the components are named index, you won't find the right one ;)

Reply:
let me rename it. it changed the function name automatically D: